rest/amino.go

```diff
@@ -69,14 +70,18 @@ func handleAminoTxsSearch(c *gin.Context, pool *pgxpool.Pool) {
 		c.AbortWithStatusJSON(400, gin.H{"error": err.Error()})
 		return
 	}
+	if height == 0 && len(events) == 0 {
```

Comment:
Seems `height` is always `0`?

Reply:
Right, didn't implement height query for Amino yet. Is there a suggested qs format for Amino?

Reply:
Or we just simply to:
```
if len(events) == 0 {
```